.gitignore

```diff
@@ -17,3 +17,5 @@ src-gen
 # IntelliJ
 .idea
 *.iml
+*.ipr
+*.iws
```

Comment:
.ipr are Intellij project files.
.iws files are dynamic web pages created sometimes when you check documentation of an object or function.

I think I found an intellij clean gradle task. These might not be necessary if I use the task.